testkit/src/api.rs

```diff
@@ -323,3 +336,47 @@ fn create_test_server(aggregator: ApiAggregator) -> TestServer {
 
     server
 }
+
+/// A convenience wrapper for Exonum node API to reduce the boilerplate code.
+pub struct ExonumNodeApi<'a> {
+    pub inner: &'a TestKitApi,
+}
+
+impl<'a> ExonumNodeApi<'a> {
+    pub fn new(api: &'a TestKitApi) -> Self {
+        Self { inner: api }
+    }
+
+    /// Asserts that the transaction with the given hash has a specified status.
+    pub fn assert_tx_status(&self, tx_hash: Hash, expected_status: &serde_json::Value) {
+        let info: serde_json::Value = self
+            .inner
+            .public(ApiKind::Explorer)
+            .query(&TransactionQuery::new(tx_hash))
+            .get("v1/transactions")
+            .unwrap();
+        if let serde_json::Value::Object(mut info) = info {
+            let tx_status = info.remove("status").unwrap();
```

Comment:
Just out of curiosity. Why do you use `remove` method instead of `get` here ? :) It's a little confusing, I suppose.

Reply:
I just copied this function from some other piece of the project:

```text
$ g grep '"status")'
examples/cryptocurrency-advanced/backend/tests/api.rs:            let tx_status = info.remove("status").unwrap();
examples/cryptocurrency/tests/api.rs:            let tx_status = info.remove("status").unwrap();
examples/timestamping/backend/tests/api.rs:        let tx_status = info.remove("status").unwrap();
```

Doesn't seem that much important, but I can replace `remove()` with `get()`, I guess.